server/subject_transform.go

```diff
@@ -114,6 +114,12 @@ func NewSubjectTransformWithStrict(src, dest string, strict bool) (*subjectTrans
 				}
 			}
 
+			if npwcs == 0 {
+				if tranformType != NoTransform {
+					return nil, &mappingDestinationErr{token, ErrorMappingDestinationFunctionWildcardIndexOutOfRange}
```

Comment:
Got to be a shorter version of this no?

`ErrorMappingDestinationFunctionWildcardIndexOutOfRange`